_posts/2017-03-27-osd-needs-better-collaboration-tools.md

```diff
@@ -14,7 +14,7 @@ Despite the rising awareness and acceptance of UX design, in particular on the w
 
 In this post, I will argue the need for better communication and collaboration platforms by looking at some of the fundamental characteristics of open source design and development and propose some starting points.
 
-The goal of OSD is to improve the user experience of open source software. Because of the nature of open source software development, to achieve this goal, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
+The goal of OSD is to improve the user experience of open source software. Because open source software development is distributed, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
```

Comment:
@studiospring Just a nitpick: Could you change the »OSD« to »Open Source Design«? Acronyms are quite difficult to understand for newcomers, and we always want to be welcoming to new people. :)